[2021-11-10 16:30:06] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:06] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":57,"packet_sent":51,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533004}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:11] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:11] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":57,"packet_sent":51,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533009}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:16] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:16] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":57,"packet_sent":51,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533014}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:21] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:21] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":57,"packet_sent":51,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533019}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:26] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:26] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":57,"packet_sent":51,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533024}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:31] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:31] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":58,"packet_sent":52,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533029}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:36] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:36] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":58,"packet_sent":52,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533034}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:41] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:41] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":64,"packet_sent":55,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533039}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:46] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:46] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":64,"packet_sent":55,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533044}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:51] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:51] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":64,"packet_sent":55,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533049}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:30:56] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:30:56] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":66,"packet_sent":56,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533054}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:31:01] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:31:01] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":57,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636533059}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
